aptos-move/e2e-move-tests/src/aggregator_v2.rs

```diff
@@ -0,0 +1,184 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{assert_success, harness::MoveHarness};
+use aptos_language_e2e_tests::account::Account;
+use aptos_types::{account_address::AccountAddress, transaction::SignedTransaction};
+use std::path::PathBuf;
+
+pub fn initialize(path: PathBuf) -> (MoveHarness, Account) {
+    let mut harness = MoveHarness::new();
+    let account = harness.new_account_at(AccountAddress::ONE);
+    assert_success!(harness.publish_package(&account, &path));
+    assert_success!(harness.run_entry_function(
+        &account,
+        str::parse("0x1::aggregator_v2_test::initialize").unwrap(),
+        vec![],
+        vec![],
+    ));
+    (harness, account)
+}
+
+pub fn check(
+    harness: &mut MoveHarness,
+    account: &Account,
+    index: u64,
+    expected: u128,
+) -> SignedTransaction {
+    harness.create_entry_function(
+        account,
+        str::parse("0x1::aggregator_v2_test::check").unwrap(),
+        vec![],
+        vec![
+            bcs::to_bytes(&index).unwrap(),
+            bcs::to_bytes(&expected).unwrap(),
+        ],
+    )
+}
+
+pub fn new(
+    harness: &mut MoveHarness,
+    account: &Account,
+    index: u64,
+    limit: u128,
+) -> SignedTransaction {
+    harness.create_entry_function(
+        account,
+        str::parse("0x1::aggregator_v2_test::new").unwrap(),
+        vec![],
+        vec![
+            bcs::to_bytes(&index).unwrap(),
+            bcs::to_bytes(&limit).unwrap(),
+        ],
+    )
+}
+
+pub fn try_add(
+    harness: &mut MoveHarness,
+    account: &Account,
+    index: u64,
+    value: u128,
+) -> SignedTransaction {
+    harness.create_entry_function(
+        account,
+        str::parse("0x1::aggregator_v2_test::try_add").unwrap(),
+        vec![],
+        vec![
+            bcs::to_bytes(&index).unwrap(),
+            bcs::to_bytes(&value).unwrap(),
+        ],
+    )
+}
+
+pub fn try_sub(
+    harness: &mut MoveHarness,
+    account: &Account,
+    index: u64,
+    value: u128,
+) -> SignedTransaction {
+    harness.create_entry_function(
+        account,
+        str::parse("0x1::aggregator_v2_test::try_sub").unwrap(),
+        vec![],
+        vec![
+            bcs::to_bytes(&index).unwrap(),
+            bcs::to_bytes(&value).unwrap(),
+        ],
+    )
+}
+
+pub fn try_sub_add(
```

Comment:
What is this method?

Reply:
This is just for the e2e move tests. The transaction first subtracts and then adds some value.